AddOns/BasicFinancials/app/src/codeunit/BasicFinancialsMgmt.Codeunit.al

```diff
@@ -0,0 +1,121 @@
+codeunit 20601 "Basic Financials Mgmt BF"
+{
+    Access = Internal;
+
+    var
+        AzureADLicensing: codeunit "Azure AD Licensing";
+        NotSupportedLicenseErr: Label 'The Basic Financials Extension can only be deployed, when at least one user has been assigned to a Basic Financials license';
+        NotSupportedLocalesErr: Label 'The Basic Financials Extension can only be deployed for the follovwing Countries: Denmark, Iceland';
+        NotSupportedUserErr: Label 'The Basic Financials Extension can only be deployed with Super User Permissions. The User %1 is not Super', Comment = '%1 = User Name';
```

Comment:
The user who deploys the Basic Financials extension must have the Super User permission set. User %1 does not have Super User permissions.